features/admin.claims.v1/components/edit/external-dialect/edit-external-claims.tsx

```diff
@@ -159,6 +167,43 @@ export const EditExternalClaims: FunctionComponent<EditExternalClaimsPropsInterf
 
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
 
+    useEffect(() => {
+        if (!attributeUri || !dialectID) {
+            return;
+        }
+        if (!SCIMConfigs.serverSupportedClaimsAvailable.includes(attributeUri)) {
+            return;
+        }
+        getServerSupportedClaimsForSchema(dialectID)
```

Comment:
Move this to a separate function and call it from here.

Reply:
Fixed in 011b0db182c8b70fc2433d301bc44890a41182be